### PR TITLE
Create /isv contact-us form

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Create /isv contact-us form [copydoc](https://docs.google.com/document/d/1zFVotFVMovmyx-Csqv_OEoB4uNMp69xOGi1WjENIU7A/edit)
- Update copy as per [copydoc](https://docs.google.com/document/d/1pcUx3oJ4ZcgRG6nGv6kzHSf9tFZZnaTK1jzecKFZxGo/edit#)

## QA

- Go to /isv compare against [copydoc](https://docs.google.com/document/d/1pcUx3oJ4ZcgRG6nGv6kzHSf9tFZZnaTK1jzecKFZxGo/edit#)
- Check the contact form opens and matches the [copydoc](https://docs.google.com/document/d/1zFVotFVMovmyx-Csqv_OEoB4uNMp69xOGi1WjENIU7A/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5005

### CODE REVIEW

File: templates/isv/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+              <li class="p-list__item">
+                <label for="email">Work email:</label>
+                <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone">Mobile/cell phone number:</label>
+                <input id="phone" name="phone" maxlength="255" type="tel" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone">Is your company a software vendor?</label>
```
Comment (mtruj013) [inline templates/isv/contact-us.html:52]: This one's supposed to be a checkbox according to the doc

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good! Just a couple of comments:

**Comment by petesfrench** [comment]
@mtruj013 Thanks for the feedback! I have kept that one line deleted as for me it is deleted in the doc.

**Comment by petesfrench** [comment]
@mtruj013 Thanks for the feedback! I have kept that one line deleted as for me it is deleted in the doc.

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Pull request merged**